packages/react/src/ReactContext.js

```diff
@@ -42,6 +42,9 @@ export function createContext<T>(
     // Secondary renderers store their context values on separate fields.
     _currentValue: defaultValue,
     _currentValue2: defaultValue,
```

Comment:
why even keep these any more instead of using "threads" on the client too? only the lack of global coordination?

Reply:
(originally set here)